react/ProductSummarySKUName.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { useCssHandles } from 'vtex.css-handles'
+import { useProductSummary } from 'vtex.product-summary-context/ProductSummaryContext'
+
+const CSS_HANDLES = ['skuNameContainer']
+
+const ProductSummarySKUName = () => {
+  const { product } = useProductSummary()
+  const handles = useCssHandles(CSS_HANDLES)
+  const skuName: string = product?.sku?.name ?? ''
+  const { productName } = product
+
+  if (!skuName || skuName === productName) {
+    return null
+  }
+
+  return (
+    <div
+      className={`${handles.skuNameContainer} flex items-start justify-center pv6`}
+    >
```

Comment:
People complain that our components styles are too opinionated. I would go to remove any default style in this case.
```suggestion
    <div className={handles.skuNameContainer}>
```